src/services/document-request-service.js

```diff
@@ -0,0 +1,65 @@
+import uniqueId from '../utils/unique-id.js';
+import DocumentRequestStore from '../stores/document-request-store.js';
+import { DOCUMENT_REQUEST_TYPE, DAY_OF_WEEK } from '../domain/constants.js';
+
+class DocumentRequestService {
+  static dependencies = [DocumentRequestStore];
+
+  constructor(documentRequestStore) {
+    this.documentRequestStore = documentRequestStore;
+  }
+
+  getDayOfWeek(date) {
```

Comment:
We do not have a `date-utils` module (only `time-utils` serving a totally different purpose), where this helper function would fit. On the other hand it can be overkill to refactor out every tiny bit of logic into some xyz-utils. So I'm 50/50 on this, and as it is only used in one place in this file, I would also not mind just simply leaving it here.

Reply:
We could make a `date-time-utils` otherwise leave it here, as it is very particular.
I'd go for the first option.

Reply:
Never mind, will go with the clean separation, `date-utils` 😊

Reply:
Done